package.json

```diff
@@ -17,11 +18,16 @@
     "react-dom": "^18.2.0"
   },
   "devDependencies": {
+    "@testing-library/jest-dom": "^6.1.4",
+    "@testing-library/react": "^14.0.0",
     "@types/node": "^20.8.8",
     "@types/react": "^18.2.32",
     "@types/react-dom": "^18.2.14",
     "@vitejs/plugin-react-refresh": "^1.3.6",
+    "@vitest/ui": "^0.34.6",
+    "jsdom": "^22.1.0",
```

Comment:
Pretty sure we don't need `jsdom`, as it'll be handled by `@testing-library/jest-dom`?